homeassistant/components/withings/const.py

```diff
@@ -10,6 +10,7 @@
 DOMAIN = "withings"
 LOG_NAMESPACE = "homeassistant.components.withings"
 PROFILE = "profile"
+CONF_PROFILE = PROFILE
```

Comment:
Should we not rename `PROFILE` to `CONF_PROFILE` instead?

Reply:
Renamed it back as I actually remove the whole option in a future PR